dicesGame/src/main/java/com/example/dices/services/DiceService.java

```diff
@@ -0,0 +1,88 @@
+package com.example.dices.services;
+
+import com.example.dices.models.DiceModel;
+import com.example.dices.repositories.IDiceRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.Random;
+
+@Service
+public class DiceService {
+    private final IDiceRepository diceRepository;
+
+    @Autowired
+    public DiceService(IDiceRepository diceRepository) {
+        this.diceRepository = diceRepository;
+    }
+
+    public static class ServiceConstants {
+        public static final int MINIMUM_DICE_SIZE = 1;
+        public static final int MAXIMUM_DICE_SIZE = 1000;
+    }
+
+    // show list of created dices
+    public List<DiceModel> getDices() {
+        return (List<DiceModel>) diceRepository.findAll();
```

Comment:
According to this instruction:
List<T>.FindAll(Predicate<T>) Método

Retrieves all elements that match the conditions defined by the specified predicate.

I understand that this is done to avoid errors, because the "findAll" method returns a collection of data of generic type called "Iterable"

Reply:
I don't think this cast is required here. We can check that today during the call.